raiden-dapp/src/views/BackupState.vue

```diff
@@ -28,27 +31,32 @@
             {{ $t('backup-state.upload') }}
           </div>
         </v-list-item-content>
-      </v-list-item> -->
+      </v-list-item>
     </v-list>
     <download-state-dialog
       :visible="downloadState"
       @cancel="downloadState = false"
     >
     </download-state-dialog>
+    <upload-state-dialog :visible="uploadState" @cancel="uploadState = false">
+    </upload-state-dialog>
```

Comment:
```suggestion
    <upload-state-dialog :visible="uploadState" @cancel="uploadState = false" />
```

Reply:
@nephix I think we probably should start enforcing that with `eslint`.

Reply:
I have created #1332 for this